tools/bench/bench.py

```diff
@@ -103,12 +112,20 @@ def main():
     parser.add_argument("--history_ints", type=int, default=0)
     parser.add_argument("--history_strings", type=int, default=0)
     parser.add_argument("--history_tables", type=int, default=0)
+    parser.add_argument("--history_images", type=int, default=0)
+    parser.add_argument("--history_images_dim", type=int, default=16)
     parser.add_argument(
         "--mode", type=str, default="online", choices=("online", "offline")
     )
     parser.add_argument("--core", type=str, default="", choices=("true", "false"))
+    parser.add_argument("--use-spawn", action="store_true")
 
     args = parser.parse_args()
+
+    # required by golang experimental client when testing multiprocessing workloads
+    if args.use_spawn:
+        multiprocessing.set_start_method("spawn")
```

Comment:
sigh :)

Reply:
heh... you dont know how sad these couple lines make me